### DISCUSSION TITLE
Getting validation error when providing an Irish postal code

### DISCUSSION THREAD

**Discussion by jlouvrier** [OP-TED/eForms-SDK#941] [Q&A] [answered]
In 1.10, adding a postal code for an Irish tenderer triggers the following Schematron validation error:
'Postcode' (BT-512-Organization-Company) is not allowed in notice type '29 – Contract award notice – general directive, standard regime' under following condition: **'Country' (BT-514-Organization-Company) does not have postcodes**

As far as I know, Irish authorities introduced postal codes about 10 years ago, with a system called Eircode. So, what is the basis for implementing such a strict rule preventing us from providing postal when it exists? I would expect Postal codes to be optional for Ireland, but not forbidden.

**Comment by YvesJo** [comment]
The postcode-country only contains countries that define and require the use of postcodes. Ireland does not require its use and is therefore not part of this list. The rules on the postcodes has been activated as part of the progressive rules integration in SDK 1.10.
This has already been reported and is currently under discussion.

**Reply by YvesJo** [reply]
Postcodes for Ireland became mandatory since the list was created. Rule will get updated in SDK 1.12. A fix will get applied to SDKs 1.11 and 1.10

**Reply by bertrand-lorentz** [reply, marked as answer]
Just a clarification for the above:
In SDK 1.12.0 the postcode will be mandatory for Ireland.
We will also include a change in upcoming patches for SDK 1.10.x and 1.11.x to allow postcodes for Ireland. They will not be mandatory, as we cannot make our rules more restrictive in patch versions.

**Comment by kgk96** [comment]
You mentioned that the fix was supposed to be applied to SDK 1.10. Has this been implemented? We are still encountering the error.

**Reply by YvesJo** [reply]
Hi,
The fix for versions 1.11.x and 1.10.x is still under process.
